src/Footer.tsx

```diff
@@ -1,21 +1,16 @@
-//import React from "react";
-//import Fade from "react-reveal/Fade";
-//import data from "../content/footer";
+import styled from 'styled-components';
 import "./styles/Footer.css";
```

Comment:
We can remove this css import now

Reply:
Since we decided on folders for components, should the footer.ts file go into the Footer folder?
```
-> src
--> Footer
---> Footer.tsx
---> footerAssets.ts
```

Reply:
Also a note that there should be an index.ts in this directory as well